docs/dietpi_tools.md

```diff
@@ -426,11 +426,63 @@ It is one of the core tools, enabling you to install or uninstall one or more [*
 
 === "DietPi survey"
 
-    This option will upload information on a anonymous base. The aggregated results of all uploads can be seen on the [DietPi survey statistics website](https://dietpi.com/survey/).  
-    Run `dietpi-survey`.
+    #### What is DietPi Survey?
+
+    DietPi Survey allows the DietPi project to obtain general information regarding your system and installed software.
 
     ![DietPi-Survey screenshot](assets/images/dietpi-survey.jpg)
 
+    - No private data is sent. No one can identify you. No IP address is obtained.
+    - This data allows the DietPi project team to focus and improve areas based on popularity. Ensuring the most common devices and software titles will receive the most support and improvements.
```

Comment:
I'm thinking if this is the right way to say it. It may sound like new or small projects won't have a chance and that we're doing like the big mobile app stores, shown/pushing what is already mainstream and dropping the alternative projects. Of course it is not unreasonable and the other way round it makes much less sense, but I personally do not have a look into the survey statistics to decide whether I should work on a software title or not, or how much time I'm gonna invest, but more based on issues/bug reports, thought-though suggestions of users and simply when I'm working around those code bits already and see that something is perfectly fine, or outdated, or does not yet follow the standards we're implementing now.

For me it is more for general interest and to make the really hard decisions: when it's about dropping support for software that is either not maintained anymore, or does not support modern frameworks/runtime systems, or support from Debian side has decreased. E.g. a current topic is software with Python 2 support, in the past it was software which did not yet support PHP7.3, while others dropped support for PHP7.0. So when it starts to be required to do nasty workarounds, custom builds or fixes and such things. That time with PHP7.0 -> PHP7.3 migration on Stretch was due to the high ownCloud usage numbers, while for the low amount of phpBB users (and there was one or two other software titles) I wouldn't have done it. And I won't start to hack Python 2 into Debian Bullseye for software that doesn't want or cannot migrate to Python 3, if there is not a notable large interest in it.

Thanks to the survey I know that we can remove the Jessie support branch and migration code, as there are no Jessie systems active anymore, which is good. I also see that dropping Stretch support this summer might become a problem, so I'm currently rethinking this plan. Similarly I now know that we need to faster offer Bullseye images for ALL SBCs, so that we do not create more Buster systems when it is already oldstable, so that the current issue with Stretch will hopefully be better two years later with Buster.

And I know that the Firefly image is actually not reasonable to keep, since there are zero reported users now. However, it is very easy to update, and I see downloads from time to time. And in such case I would love to have more than 14% survey participants to be more certain about the actual number, since it's statistically totally possible that actually there are 20 or more Firefly users. It also could be zero, and for sure it's nothing compared to other SBCs, but if 5 interactive minutes (I use unattended runtime for other things), hopefully soon a completely automated build, serve accumulated 20 people with an up-to-date server image, it's fine for me 😉.

Long text, sorry, I just want to give some insights about how I personally use the survey, but I'm not sure how to put this best into one sentence, or if other contributors use it differently 😉.

Reply:
OK, changed the text.

@MichaIng: Please also review the sentence describing "Data transmission events": Is it `dietpi-software` and `dietpi-update` or also `apt upgrade` resp. `apt dist-upgrade` resp. `apt full-upgrade` resp. `apt-get ...` etc?

Reply:
It's at the end of `dietpi-software` installs and `dietpi-update`, so all correct 👍.